tests/ut/ir/operators/test_alloc_tile_op.py

```
@@ -0,0 +1,127 @@
+            ir.ConstInt(4096, DataType.INT64, span),
+        ],
+        span,
+    )
+    assert not isinstance(region.type, ir.TileType)
+    assert isinstance(_make_alloc_tile().type, ir.TileType)
+
+
+def test_alloc_tile_rejects_wrong_arg_count():
+    span = ir.Span.unknown()
+    base = ir.Var("mem_vec_0", ir.PtrType.get(), span)
+    with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)